.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,23 +4,63 @@ about: Create a report to help us improve
 labels: 'bug'
 ---
 
+**Bug was detected in one of the following services:**
```

Comment:
I'd have kept this section right before the **Environment** section as I feel that the bug description itself is the primary piece of information for a bug report and many external users may not be able to pin point that issue to a specific component.